website/api/duk_to_summary.yaml

```diff
@@ -0,0 +1,26 @@
+name: duk_to_summary
+
+proto: |
+  const char *duk_to_summary(duk_context *ctx, duk_idx_t idx);
+
+stack: |
+  [ ... val! ... ] -> [ ... summary! ... ]
+
+summary: |
+  <p>Replace the value at <code>idx</code> with a readable summary of the value
+  in string form.  If <code>idx</code> is invalid, throws an error.</p>
+
```

Comment:
Maybe one or two sentences about the intent and specific guarantees:

- Summary is a short string with a maximum length, strings and buffer values are truncated.
- Coercing a value to a summary does not cause side effects such as calling `.toString()` methods or such. Garbage collection side effects (like running a finalizer) are possible.

A table of example summary values for API types, and perhaps a few common objects, would be quite useful when browsing the API docs.